justfile

```
@@ -28,3 +28,7 @@ check-commits:
   else
     echo "Not on main or haven't committed yet."
   fi
+
+# Update all dependencies in lockfile
+update-deps:
+  poetry update --lock
```

```suggestion
  poetry update
```

I believe with the `--lock` command it only updates the lock file, but not actually installs anything new https://python-poetry.org/docs/cli/#update